Mixpanel/MixpanelInstance.swift

```diff
@@ -621,12 +621,14 @@ extension MixpanelInstance {
                 self.people.distinctId = distinctId
             }
             if !self.people.unidentifiedQueue.isEmpty {
+                objc_sync_enter(self)
                 for var r in self.people.unidentifiedQueue {
                     r["$distinct_id"] = self.distinctId
                     self.people.peopleQueue.append(r)
                 }
                 self.people.unidentifiedQueue.removeAll()
-                Persistence.archivePeople(self.flushPeopleQueue + self.people.peopleQueue, token: self.apiToken)
+                objc_sync_exit(self)
+                Persistence.archivePeople(self.people.flushPeopleQueue + self.people.peopleQueue, token: self.apiToken)
```

Comment:
swift is non-atomic so we will need any copy (set) be under a thread-safe encapsulation